### ISSUE TITLE
Project import tree improvements

### ISSUE THREAD

**Issue by drewnoakes** [dotnet/project-system#5702]
Following #4805 and #5639 there are some pending items we are considering for the project import tree:

- [ ] Browse object showing path, label, group label
- [ ] Option to show these at all times
- [ ] Include imported files in "Find in Files"
- [ ] (Maybe) track "Show all files" state via Dataflow

**Comment by drewnoakes** [comment]
- [ ] While the import tree is visible, drag/dropping the project's `.csproj.user` file into the editor gives:

  > 'ClassLibrary1.csproj.user' cannot be opened. The file is not supported.

**Comment by drewnoakes** [comment]
- [ ] Review context menu on the "Imports" node

**Comment by drewnoakes** [comment]
- [ ] Review whether ordering of children in tree reflects ordering of imports, including in the case of updates that change order
- [ ] Investigate the 'Search within file contents' option of Solution Explorer's search:
   ![image](https://user-images.githubusercontent.com/350947/70571401-efa72b80-1bf1-11ea-9396-b2c246667d05.png)

**Comment by davkean** [comment]
@drewnoakes This is marked tracking but it actually has work inside of it.

**Comment by drewnoakes** [comment]
@shyamnamboodiripad I looked into the ordering of tree items. They're being displayed alphabetically. Once #5736 lands, they'll be displayed in the correct evaluation order.

**Comment by shyamnamboodiripad** [comment]
> They're being displayed alphabetically. Once #5736 lands, they'll be displayed in the correct evaluation order.

@drewnoakes Sounds great - Thank you!

**Comment by davkean** [comment]
@drewnoakes Can you split this up into individual issues that folks might be able to bite off under "Help-Wanted"?

**Comment by drewnoakes** [comment]
I will once my current bout of work on it slows down.

**Comment by davkean** [comment]
Find in Files:

- It appears that the new search only searches folders that return GUID_ItemType_PhysicalFolder for VSHPROPID_TypeGuid and does not respect VSHPROPID_IsNonSearchable. The imports node is a virtual folder and is not considered GUID_ItemType_PhysicalFolder, nor do we want to for it to be considered that as that brings other behavior along with it, such as provided a file name from ProjectItem.FileNames.
